### ISSUE TITLE
Automate pip package release process

### ISSUE THREAD

**Issue by mjenczmyk** [luk6xff/Packt-Publishing-Free-Learning#187]
Hi @luk6xff,

I've been trying to [automate pip package release process](https://github.com/actions/starter-workflows/blob/master/ci/python-publish.yml) with GitHub actions, so I woudn't have to make it myself. Unfortunately this requires me to provide my PyPi credentials (that should remain secret) and I don't have access to this repository's settings.

There are two solutions: either you may create a new `packt-cli` organization on GitHub, give ownership of this repository to this organization and grant me sufficient permissions inside this organization or I may grant you maintainer role of [PyPi package](https://pypi.org/project/packt/), than we'd use your credentials in this repository.

What do you think? Are we going to do this?

**Comment by mjenczmyk** [comment]
@luk6xff Which way are we going to go?

**Comment by mjenczmyk** [comment]
@luk6xff Can I remind you about this issue?

**Comment by luk6xff** [comment]
Hello, welcome back after long time, I was mega busy with other projects for last year and had no time for checking the github inbox. Sure I can create such organization, or maybe You can create and we shall move this repository to your account ? As you've been taking care of this repo, you could create a fork and provide support for it on your own rules . What do you think  ?

**Comment by mjenczmyk** [comment]
Hi,

I've invited you to the organization as an owner - I want you to have a control over the repository. If you don't mind let's move the repository there (preferably in a way that will redirect from the old URL to the new one - I don't use GitHub on everyday so I'm not sure how to do that). I believe that than I'll be able to automate release process.

BTW, should I close the #186 issue?

**Comment by luk6xff** [comment]
@mjenczmyk  That's great!, Thanks a lot :) , I have moved the repo  right now :) https://github.com/packt-cli

**Comment by luk6xff** [comment]
Of course, please close #186 and #200 . Gitlab is good but on github we will have much more visitors and users of the script :)

**Comment by mjenczmyk** [comment]
Thanks, I can see that https://github.com/luk6xff/Packt-Publishing-Free-Learning is redirecting to the new URL.

Yeah, I use GitLab because I'm not a great fan of Microsoft, but the audience is bigger on the GitHub indeed. I'll close those issues, in my free time I'll have a look how to automate the release here - I don't like to do it manually `git push` should be everything I need to release the package ;)
